api/serverless.yml

```diff
@@ -32,20 +30,18 @@ functions:
     name: oss-nameguard-${self:custom.stage}
     memorySize: 1769
     timeout: 10
-    versionFunctions: true
```

Comment:
This was not a valid property

Reply:
You mean that this is `true` by default right?

Reply:
![image](https://github.com/user-attachments/assets/bb03034f-0c76-41d2-ab60-5eaee879a364)

This property was causing the error above.